### PR TITLE
Remove usage of prefix increment operator.

### PR DESCRIPTION
**Pull request by markstory**
This operator is less frequently used and can be harder to understand than the more common suffix operator.

Refs #12019

### CODE REVIEW

File: src/Utility/Text.php [hunk 1]
```diff
@@ -138,7 +138,8 @@ public static function tokenize($data, $separator = ',', $leftBound = '(', $righ
                         }
                     }
                 }
-                $offset = ++$tmpOffset;
+                $tmpOffset += 1;
+                $offset = $tmpOffset;
```
Comment (saeideng) [inline src/Utility/Text.php]: why not 
```php
$offset = $tmpOffset+1;
```

Comment (dereuromark) [reply src/Utility/Text.php]: That is sth different.

Comment (saeideng) [reply src/Utility/Text.php]: @dereuromark  I mistake :+1:

File: src/View/Helper/HtmlHelper.php [hunk 2]
```diff
@@ -982,10 +982,11 @@ protected function _renderCells($line, $useCount = false)
             }
 
             if ($useCount) {
+                $i += 1;
```
Comment (dereuromark) [inline src/View/Helper/HtmlHelper.php]: Should we then use `$i++` for consistency?

**Pull request merged**